src/esp/sim/Simulator.h

```diff
@@ -251,6 +254,14 @@ class Simulator {
    */
   std::vector<int> getExistingObjectIDs(int sceneID = 0);
 
+  /**
+   * @brief Get the SceneNode of a physics object instanced in a physical scene.
+   * See @ref esp::physics::PhysicsManager::getExistingObjectIDs.
+   * @param objectID The ID of the object identifying it in @ref
+   * @return A reference to the SceneNode
+   */
+  esp::scene::SceneNode& getObjectNode(const int objectId);
```

Comment:
Isn't this the same as getObjectSceneNode?

Reply:
Thanks Erik! I added this function because I couldn't find sceneID anywhere in viewer, which is a parameter to getObjectSceneNode. But now I see that there are some functions in Simulator which have sceneID = 0 by default, so we could do that for getObjectSceneNode. I'll make that change!